### PR TITLE
Support for Client ID and secret in notm and keyworker

### PR DESCRIPTION
**Pull request by Mjwillis**
Currently just dev environments  - Will need to do stage, prod, and preprod later

### CODE REVIEW

File: notm/webapp-config.template.json [hunk 1]
```diff
@@ -18,6 +18,12 @@
         "USE_API_GATEWAY_AUTH": {
```
Comment (glenjamin) [inline notm/webapp-config.template.json]: Might be a good opportunity to sack off this file and replace it with the native terraform support for handling config.

File: keyworker-ui/dev/main.tf [hunk 2]
```diff
@@ -22,7 +22,7 @@ resource "azurerm_app_service_plan" "omic-ui" {
 
   sku {
     tier     = "Standard"
-    size     = "S1"
+    size     = "S2"
```
Comment (glenjamin) [inline keyworker-ui/dev/main.tf]: Is this scaled up for a reason?

File: keyworker-ui/dev/main.tf [hunk 3]
```diff
@@ -58,6 +60,7 @@ data "external" "vault" {
     vault = "${azurerm_key_vault.vault.name}"
     api_gateway_private_key = "api-gateway-private-key"
     api_gateway_token = "api-gateway-token"
+    api_client_secret = "api-client-secret"
```
Comment (glenjamin) [inline keyworker-ui/dev/main.tf]: Have you already populated this in the keyvaults or do we need to do that too?

File: keyworker-ui/dev/main.tf (hunk 2 above)
Comment (Mjwillis) [reply keyworker-ui/dev/main.tf]: Yes it temporary, I want to see if giving it a bigger box will surface the timeout issue (its probably a memory leak - but the devs can't see it)

File: keyworker-ui/dev/main.tf (hunk 3 above)
Comment (Mjwillis) [reply keyworker-ui/dev/main.tf]: I've done it

**Comment by mattops** [comment]
Looks good.

**Pull request merged**